sentry-contrib-native-sys/Cargo.toml

```diff
@@ -1,9 +1,12 @@
 [package]
-name = "sentry-native-sys"
+name = "sentry-contrib-native-sys"
 version = "0.1.0"
 authors = ["daxpedda <daxpedda@gmail.com>"]
 edition = "2018"
 links = "sentry-native"
 
 [build-dependencies]
 anyhow = "1"
+
+[features]
+custom-transport = []
```

Comment:
too bad cargo does not support enum-type features, to choose the native backend that you want.

Reply:
That would be nice, but this mostly works. ;)

Reply:
I reversed it in https://github.com/daxpedda/sentry-contrib-native/commit/1bbf83ed7b561fad82d05eb07bbdcf71c0f83589 and called it `default-transport`.

When we add backend choice, we can make those features mutually exclusive and check for it in `build.rs`, best I can come up with as an alternative to an enum-type feature.